dask/array/tests/test_fft.py

```diff
@@ -73,6 +126,21 @@ def test_irfft():
     assert_eq(irfft(darr2, axis=0), np.fft.irfft(nparr, axis=0))
 
 
+@pytest.mark.parametrize(
+    "funcname",
+    ["irfft2", "irfftn"]
+)
+def test_irfft2n(funcname):
+    da_fft = getattr(dask.array.fft, funcname)
+    np_fft = getattr(np.fft, funcname)
+    assert_eq(da_fft(darr3),
+              np_fft(nparr))
+    assert_eq(da_fft(darr3, axes=(1, 0)),
+              np_fft(nparr, axes=(1, 0)))
+    assert_eq(da_fft(darr3, (8, 9), axes=(1, 0)),
+              np_fft(nparr, (8, 9), axes=(1, 0)))
```

Comment:
Can you provide an example of a test that couldn't be done in this way?  Almost all dask.array tests are of the form:

1.  Do something with numpy
2.  Do the equivalent thing with dask.array
3.  Call assert_eq on the results (which should check many things)

This *usually* works, but you're right that sometimes we need to break this pattern.  I think that the question you're posing is if this is one of those cases.  I don't yet understand what would break about the pattern above for ifft functions.